tox.ini

```diff
@@ -763,7 +763,9 @@ commands =
     bash -c 'rm packages/grid/helm/syft/templates/* || true'
 
     bash -c 'cd packages/grid && \
-        devspace deploy --skip-build --no-warn --kube-context "k3d-build" --render 2>/dev/null | python3 helm/helm.py'
+        [[ -n "$CONTAINER_REGISTRY" ]] && REGISTRY_FLAG="--var CONTAINER_REGISTRY=$CONTAINER_REGISTRY" || REGISTRY_FLAG="" && \
+        echo $REGISTRY_FLAG && \
+        devspace deploy --skip-build --no-warn ${REGISTRY_FLAG} --kube-context "k3d-build" --render 2>/dev/null | python3 helm/helm.py'
```

Comment:
If `CONTAINER_REGISTRY` env is passed, then we add `--var CONTAINER_REGISTRY=$CONTAINER_REGISTRY` to devspace deploy command. This will generate helm charts pointing to Azure container registry URL